### PR TITLE
PST03A-v2.2.5 Support

### PR DESCRIPTION
**Pull request by echauvet**
This device looks like this ;
PST03A

### CODE REVIEW

File: zhaquirks/philips/zhadimmablelight.py [hunk 1]
```diff
@@ -39,11 +39,14 @@ class ZHADimmableLight(CustomDevice):
...
             (PHILIPS, "LWO001"),
             (PHILIPS, "LWO003"),
+            (PHILIPS, "LWU001"),
             (PHILIPS, "LWV001"),
             (SIGNIFY, "LWA004"),
             (SIGNIFY, "LWA005"),
+            (SIGNIFY, "LWA007"),
             (SIGNIFY, "LWO001"),
             (SIGNIFY, "LWO003"),
+            (SIGNIFY, "LWU001"),
```
Comment (dmulcahey) [inline zhaquirks/philips/zhadimmablelight.py:49]: are these changes intentional? they aren't related to the device you are adding support for...

Comment (echauvet) [reply zhaquirks/philips/zhadimmablelight.py:49]: You are right. I pull the dev. My mistake. But I think this commit under the right PR is OK. I'm working around ZB philips OTA too.

Comment (echauvet) [reply zhaquirks/philips/zhadimmablelight.py:49]: An action is needed?

Comment (echauvet) [reply zhaquirks/philips/zhadimmablelight.py:49]: I'm closing this PR as it seems to make trouble. I will publish a new one without issues in one commit
